baseplate/experiments/providers/__init__.py

```diff
@@ -87,11 +94,12 @@ def parse_experiment(config):
         )
         version = None
     now = time.time()
-    if now < start_ts or now > stop_ts:
-        return ForcedVariantExperiment(None)
 
     enabled = config.get("enabled", True)
-    if not enabled:
+    if now < start_ts or now > stop_ts:
```

Comment:
I'd check for enabled first-a disabled experiment might lack one or both of start/stop.

Reply:
I believe that the initial intention was that it was always required.